src/main/java/com/digital/home/service/DailyTransactionService.java

```diff
@@ -0,0 +1,66 @@
+package com.digital.home.service;
+
+import com.digital.home.model.*;
+import com.digital.home.repository.LabourDetailsRepository;
+import com.digital.home.repository.LabourMasterRepository;
+import com.digital.home.repository.MaterialDetailsRepository;
+import com.digital.home.repository.MaterialMasterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class DailyTransactionService {
+
+    @Autowired
+    private MaterialDetailsRepository materialDetailsRepository;
+
+    @Autowired
+    private LabourDetailsRepository labourDetailsRepository;
+
+    @Autowired
+    private MaterialMasterRepository materialMasterRepository;
+
+    @Autowired
+    private LabourMasterRepository labourMasterRepository;
+
```

Comment:
Can you have two different methods to insert Labour details and Material details as in mobile we cannot enter everything and post in one transaction

Reply:
Added two different methods as suggested.